Sources/_RegexParser/Regex/Parse/SourceLocation.swift

```diff
@@ -77,10 +77,10 @@ extension Source {
     }
   }
 
-  /// Located value: a value wrapped with a source range
+  /// A value wrapped with a source range.
```

Comment:
QUERY: This is a best-guess rewrite of what the abstract is trying to say.

Reply:
This is correct. `Located` lets us track the source of each piece of a regex, which can then be included in the error output.